pubsub.go

```diff
@@ -94,6 +94,10 @@ func (p *PubsubValueStore) PutValue(ctx context.Context, key string, value []byt
 		p.mx.Unlock()
 	}
 
```

Comment:
We should remove the above bootstrapping lines if we're going to do it this way.

Reply:
@Stebalien done. #28 is also now glaringly obvious now, since we see that the context passed in is just ignored. If you think it makes sense I could wrap the insides of this function (Subscribe + Publish) in a gofunc + channel wait wrapper, but I'm inclined to just leave that until we implement the Discovery in PubSub, per #28 .

Reply:
I'd leave it as-is. _Subscribe_ shouldn't block anyways.